lychee-lib/src/client.rs

```diff
@@ -173,39 +184,50 @@ pub struct ClientBuilder {
     exclude_mail: bool,
 
     /// Maximum number of redirects per request before returning an error.
+    ///
+    /// Defaults to [`DEFAULT_MAX_REDIRECTS`].
     #[builder(default = DEFAULT_MAX_REDIRECTS)]
     max_redirects: usize,
 
     /// Maximum number of retries per request before returning an error.
+    ///
+    /// Defaults to [`DEFAULT_MAX_RETRIES`].
     #[builder(default = DEFAULT_MAX_RETRIES)]
     max_retries: u64,
 
     /// User-agent used for checking links.
     ///
-    /// *NOTE*: This may be helpful for bypassing certain firewalls.
+    /// Defaults to [`DEFAULT_USER_AGENT`].
+    ///
+    /// # Notes
+    ///
+    /// This may be helpful for bypassing certain firewalls.
     // Faking the user agent is necessary for some websites, unfortunately.
     // Otherwise we get a 403 from the firewall (e.g. Sucuri/Cloudproxy on ldra.com).
     #[builder(default_code = "String::from(DEFAULT_USER_AGENT)")]
     user_agent: String,
 
     /// When `true`, accept invalid SSL certificates.
     ///
-    /// ## Warning
+    /// # Warning
     ///
-    /// You should think very carefully before using this method. If
-    /// invalid certificates are trusted, any certificate for any site
-    /// will be trusted for use. This includes expired certificates. This
-    /// introduces significant vulnerabilities, and should only be used
-    /// as a last resort.
+    /// You should think very carefully before allowing invalid SSL
+    /// certificates. It will accepts any certificate for any site to be trusted
+    /// including expired certificates. This introduces significant
+    /// vulnerabilities, and should only be used as a last resort.
+    // TODO: We should add a warning message in CLI. (Lucius, Jan 2023)
```

Comment:
Not a big fan of adding usernames to `TODO`s.

Reply:
it helps to track who added the idea without the need to use git blame.

Reply:
Admittedly, it's a rather minor thing but I'd still prefer if we could remove it.   
`git blame` works well and the usernames in comments cause some mental overhead with little benefit, at least to me. I also don't know of any bigger Rust project that uses them. Besides that, it shouldn't matter who added a `TODO` as long as it's relevant to the section of the code.  We should track ideas in Github discussions and issues in Github issues.

Reply:
https://github.com/rust-lang/rust/blob/4a04f252f91b2ccef6fc16817b3e7a3aeda703b6/compiler/rustc_codegen_gcc/src/lib.rs#L2-L6

this is a well known convention and it's even in our compiler.

Reply:
Still not convinced (for the reasons above), but I'm fine with keeping it in.